osgar/drivers/replay.py

```diff
@@ -42,6 +42,8 @@ def run_input(self):
             data = deserialize(data_raw)
             # TODO reuse timestamp
             sub_channel = self.pins[channel]
+            if ":" in sub_channel:
+                sub_channel = sub_channel.split(":")[0]
```

Comment:
well, it is still not clear to me ... but I surely mixed up "replay driver" and "osgar.replay". If this works better for you let's use it.